beacon_node/network/src/sync/network_context.rs

```diff
@@ -46,6 +46,10 @@ pub enum RangeRequestId {
     },
 }
 
+/// 0: expected blob count
+/// 1: block slot
+pub type DownloadedBlockSummary = (/* expected blob count */ usize, Slot);
```

Comment:
a struct might be nice here
```
pub struct DownloadedBlockSummary {
    expected_blob_count: usize,
    block_slot: Slot
}
```

Reply:
Removed the type / struct altogether by just passing the full block around in 96e2363d721f55d79f66f98f15fd61a854c7dc62